thunder/core/jit_ext.py

```diff
@@ -1347,6 +1347,18 @@ def bind_inputs(name, trace, input_vars, input_proxies):
     trace.args = input_proxies
 
 
+def _get_process_group_from(*fn_and_args):
+    # `ddp` and `fsdp` transforms add attribute `procses_group_for_ddp`
+    # on the Module that they wrap. This module could be passed to `thunder.jit`
+    # as the function to be jitted or as an argument of the function to be jitted.
+    # NOTE: `ddp` and `fsdp` use default process group. So finding the first
+    #       occurence is sufficient.
+    for fn_or_arg in fn_and_args:
+        pg = getattr(fn_or_arg, "process_group_for_ddp", None)
+        if pg is not None:
+            return pg
```

Comment:
nit

```suggestion
    for fn_or_arg in fn_and_args:
        if (pg := getattr(fn_or_arg, "process_group_for_ddp", None)) is not None:
            return pg
```

Reply:
Let's not do this. We're not short on space and it has more non-whitespace characters.